### DISCUSSION TITLE
如何显示单个页面的大纲

### DISCUSSION THREAD

**Discussion by Mojicode** [toeverything/AFFiNE#2005] [General]
如题，请教如何显示单个页面的大纲视图，这样浏览起来比较清晰方便？

**Comment by HeJiachen-PM** [comment]
这个功能暂时未被支持。感谢您的反馈。

Mojicode ***@***.***> 于 2023年4月18日周二 14:24写道：

**Reply by Mojicode** [reply]
> 这个功能暂时未被支持。感谢您的反馈。 Mojicode ***@***.***> 于 2023年4月18日周二 14:24写道：
> […](#)
> 如题，请教如何显示单个页面的大纲视图，这样浏览起来比较清晰方便？ — Reply to this email directly, view it on GitHub <#2005>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AS5AYR25NVP2WS5NQ5QOJW3XBYXQJANCNFSM6AAAAAAXCEVJU4> . You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

请问有计划支持吗

**Reply by HeJiachen-PM** [reply]
有的，预计在0.8.0之前支持。

**Comment by HeJiachen-PM** [comment]
A toc of docs is planned to be implemented before 0.8.0
